bin/oref0-pump-loop.sh

```diff
@@ -155,7 +155,7 @@ function smb_reservoir_before {
 
 # check if the temp was read more than 5m ago, or has been running more than 10m
 function smb_old_temp {
-    (find monitor/ -mmin +5 -size +5c | grep -q temp_basal && echo temp_basal.json more than 5m old) \
+    (file_is_recent_and_min_size monitor/temp_basal.json && echo temp_basal.json more than 5m old) \
```

Comment:
This line was incorrect - it inverts the age part of the test. Reverted in later commit.